### PR TITLE
Fix dotnet format argument

### PR DESCRIPTION
**Pull request by juchom**
## What does this PR do?

dotnet format replace `--check` with `--verify-no-changes`

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -41,4 +41,4 @@ jobs:
     - name: Install dotnet-format
       run: dotnet tool install -g dotnet-format
     - name: Check with dotnet-format
-      run: dotnet format --check Meilisearch.sln
+      run: dotnet format --verify-no-changes Meilisearch.sln
```
Comment (brunoocasali) [inline .github/workflows/tests.yml:44]: Do you know why this change now?

Comment (brunoocasali) [reply .github/workflows/tests.yml:44]: I had the same problem here: https://github.com/meilisearch/meilisearch-dotnet/pull/233 and I "solved" by fixing the version of the action actions/setup-dotnet@v1.

Comment (juchom) [reply .github/workflows/tests.yml:44]: dotnet format has been integrated with .net 6 sdk. They replaced `check` by `verify-nochanges` in order to have names that better fit with dotnet cli.

Comment (juchom) [reply .github/workflows/tests.yml:44]: Actually, you should keep your change and close this one. Building and testing are done against 3.1.

Comment (brunoocasali) [reply .github/workflows/tests.yml:44]: Ok, good to know, thanks for sharing the knowledge!

**Comment by brunoocasali** [comment]
Closed, since we will close the issue after the https://github.com/meilisearch/meilisearch-dotnet/pull/233 get merged.
